mhs/mhs/common/state/work_description.py

```diff
@@ -0,0 +1,155 @@
+import json
+import utilities.integration_adaptors_logger as log
+import datetime
+
+logger = log.IntegrationAdaptorsLogger('STATE_MANAGER')
+
+
+class MessageStatus:
```

Comment:
I actually used enum at first, but it didn't serialize to json without a custom serialization method which seemed a bit long winded/ overkill considering I wasn't fully sure how it was going to be used in the code yet, if we find it might be useful/cleaner to define it as an enum I'm happy update it

Reply:
I was about to suggest `IntEnum`, but then that wouldn't be that readable in the database. Only other thing I can think of is to just use the value only when serialising to json, but meh, we can look at this later in the tasks that use this code.

Reply:
Since writing the above comment, I found what I was looking for: According to [here](https://docs.python.org/3.7/library/enum.html#others), you can create a string enum pretty simply. And I expect that that would serialise in the manner you want it to. Then you can use the [`unique`](https://docs.python.org/3.7/library/enum.html#ensuring-unique-enumeration-values) decorator as well.